encoders/firebase-decoders-json/src/json/java/com/google/firebase/decoders/json/JsonDataDecoderBuilderContext.java

```diff
@@ -87,8 +86,7 @@
     return convertGenericListToArray(list, arrayToken);
   }
 
-  private static <T, E> T convertGenericListToArray(
-      List<Object> list, TypeToken.ArrayToken<T> arrayToken) {
+  static <T, E> T convertGenericListToArray(List<Object> list, TypeToken.ArrayToken<T> arrayToken) {
```

Comment:
this method works like a utility method, does it worth the efforts to move it into a helper file?